sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -18,22 +22,66 @@ You may also refer to [Authenticating a user account interactively in the browse
 
 ## Authenticating server side applications
 
-For server side applications we provide options that vary from minimal configuration with sensible defaults using the `DefaultAzureCredential` to more specialized credentials that can support your specific scenario. Please feel free to navigate to the specific credentials below to explore what we offer:
+For server side applications we provide options that vary from minimal configuration with sensible defaults using the `DefaultAzureCredential` to more specialized credentials that can support your specific scenario.
 
-- [Authenticating with `DefaultAzureCredential`](#authenticating-with-defaultazurecredential)
-- [Authenticating a user assigned managed identity with `DefaultAzureCredential`](#authenticating-a-user-assigned-managed-identity-with-defaultazurecredential)
-- [Authenticating a user account interactively in the browser](#authenticating-a-user-account-interactively-in-the-browser)
-- [Authenticating a service principal with a client secret](#authenticating-a-service-principal-with-a-client-secret)
-- [Authenticating a service principal with a client certificate](#authenticating-a-service-principal-with-a-client-certificate)
-- [Authenticating a service principal with environment credentials](#authenticating-a-service-principal-with-environment-credentials)
-- [Authenticating a user account with device code flow](#authenticating-a-user-account-with-device-code-flow)
-- [Authenticating a user account with username and password](#authenticating-a-user-account-with-username-and-password)
-- [Authenticating a user account with auth code flow](#authenticating-a-user-account-with-auth-code-flow)
-- [Authenticating a user account with Azure CLI](#authenticating-a-user-account-with-azure-cli)
-- [Authenticating a user account with Visual Studio Code](#authenticating-a-user-account-with-visual-studio-code)
-- [Authenticating in Azure with managed identity](#authenticating-in-azure-with-managed-identity)
+- To get started, you can always rely on interactive authentication of your user account.
+- As you develop your application, you may want to first sign in using the developer tools like Azure CLI or Visual Studio Code to avoid signing in interactively every time you run your application.
+- As you deploy your application to Azure App Service or run it in a virtual machine, you may want to make use of [Managed Identity](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview).
 
-### Authenticating with `DefaultAzureCredential`
+You can chain multiple credentials together so that they are tried sequentially until one of them succeeds.
```

Comment:
How common is creating a custom chained credential (if we know) - this content is pretty high up so I was curious. We do have a section on it at the bottom of this doc. Should we pull that up or link to it from here?

Reply:
@schaabs has on multiple occasions mentioned that when DefaultAzureCredential doesnt work for certain scenarios, we prefer users to chain their own credentials rather than us trying to show horn every credential into DefaultAzureCredential. Since we want to advertise the "use diff stuff without changing your code!", I found it worth mentioning here. It also served as a good intro to DefaultAzureCredential itself.

I wouldnt pull up the entire section on chaining credential up here, but adding a link to it would be good

Reply:
Added link to chaining credentials in https://github.com/Azure/azure-sdk-for-js/pull/14920/commits/e80064050fb4900b9a687dffa8856b9780c2d014